src/core/exports/export-credentials-k8s.ts

```diff
@@ -0,0 +1,35 @@
+type K8s = {
+    "K8S_CLUSTER": string;
+    "K8S_USER": string;
+    "K8S_SERVER_URL": string;
+    "K8S_NAMESPACE": string;
+    "K8S_TOKEN": string;
+    "K8S_EXPIRATION": string;
+};
+
+export const exportK8sCredentials = [
+    {
+        id: "bash",
+        label: "bash",
+        fileName: "init-kub.sh",
+        text: (c: K8s) =>
+            `#!/bin/bash
+kubectl config set-cluster ${c.K8S_CLUSTER} --server=${c.K8S_SERVER_URL}
+kubectl config set-credentials ${c.K8S_USER} --token ${c.K8S_TOKEN} 
+kubectl config set-context ${c.K8S_CLUSTER} --user=${c.K8S_USER} --cluster=${c.K8S_CLUSTER} --namespace=${c.K8S_NAMESPACE}
+kubectl config use-context ${c.K8S_CLUSTER}
+`,
+    },
+    {
+        id: "shell",
+        label: "shell",
+        fileName: "init-kub.sh",
+        text: (c: K8s) =>
+            `#!/bin/sh
+kubectl config set-cluster ${c.K8S_CLUSTER} --server=${c.K8S_SERVER_URL}
+kubectl config set-credentials ${c.K8S_USER} --token ${c.K8S_TOKEN} 
+kubectl config set-context ${c.K8S_CLUSTER} --user=${c.K8S_USER} --cluster=${c.K8S_CLUSTER} --namespace=${c.K8S_NAMESPACE}
+kubectl config use-context ${c.K8S_CLUSTER}
+`,
+    },
+];
```

Comment:
We should not create a new directory in the `src/core` directory.  
There should be a new slice use case for this feature.  
(e.g. a new slice `src/core/usecases/initScriptsGenerator.ts`? )  
The role of the `usecases` are to `spoon-feed` to the UI everything it needs to do its job.   
The goal is for the UI to be as dumb as possible.  
In your implementation the UI is responsible for getting the region, extracting the kubernetes url, not displaying the tab if there is not such url, calling this function to generate the scrip.  
All this logic should be moved hover to the core.